wave/src/WaveServiceProvider.php

```diff
@@ -148,6 +148,76 @@ protected function loadBladeDirectives(){
             return '{!! view("wave::checkout")->render() !!}';
         });
 
+        // admin Directives
+
+        Blade::directive('admin', function () {
+            return "<?php if (!auth()->guest() && auth()->user()->hasRole('admin')) { ?>";
+        });
+
+        Blade::directive('notadmin', function () {
```

Comment:
This naming of **not**role is weird. `notadmin` sounds like it should be another directive targeting everybody that's not an admin. Meaning every role below an admin role, for instance, a guest, customer, or a beta member. Not an else statement.

Why divert from the Laravel standard? They make it easy to follow the standard with the [custom if statement](https://laravel.com/docs/9.x/blade#custom-if-statements) `Blade::if` function.

Noticed just now that Wave uses **not**role, still don't like it so my comment stays. 🙂

```php
Blade::if('admin', function () {
    return !auth()->guest() && auth()->user()->hasRole('admin');
});

@admin
    <!-- The user has role of admin... -->
@elseadmin
    <!-- The user does NOT have role of admin... -->
@endadmin
```